src/Utils.cpp

```diff
@@ -28,7 +28,7 @@ bool isValidInertiaMatrix(Eigen::Matrix3d inertia) {
   bool isValid = true;
 
   Eigen::VectorXcd eigenvalues = inertia.eigenvalues();
-  isValid &= (eigenvalues.real().minCoeff() >= 0.0);
+  isValid &= (eigenvalues.real().minCoeff() > -1e-9);
```

Comment:
can we add a comment here why this is necessary?

Reply:
done